src/cobble/git_version.py

```diff
@@ -310,14 +310,17 @@ def all_first_base_branch_commits(self) -> List[Commit]:
 
     @property
     def head_branch_name(self) -> str:
-        return self.git_client.head_branch_name()
+        try:
+            return self.git_client.head_branch_name()
+        except subprocess.CalledProcessError:
```

Comment:
Maybe a comment here. Technically this try-except is too broad since it catches any CalledProcessError.  We could either parse the expected output or just comment here that this may accidentally catch other errors that could be dumped at this point>

Reply:
I made this a bit narrower and check the output.